engine/xml-config/src/main/resources/job.xsd

```diff
@@ -5,19 +5,26 @@
 
 	<element name="job">
 		<complexType>
-			<sequence>
-				<element name="job-metadata" type="ab:jobMetadataType"
-					minOccurs="0" maxOccurs="1" />
-				<element name="source" type="ab:sourceType" minOccurs="0"
-					maxOccurs="1" />
-				<element name="transformation" type="ab:transformationType"
-					minOccurs="0" maxOccurs="1" />
-				<element name="analysis" type="ab:analysisType" minOccurs="1"
-					maxOccurs="1" />
-			</sequence>
+			<complexContent>
+				<extension base="ab:jobType">
+				</extension>
+			</complexContent>
```

Comment:
This seems a bit overdone: If you want to make an element of jobType, ```<element name="job" type="ab:jobType"/>``` should be enough.